### PR TITLE
Uhf 7449 announcement sync v3

### PR DESCRIPTION
**Pull request by rpnykanen**
# Announcement sync V3 [UHF-7449](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7449)

## What was done
Added new block which utilises external entities
Added new external entity type for global announcements

## How to install
* [Review this etusivu PR first](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/290)
* Make sure you have this line in your `local.settings.php`: `$config['helfi_announcement.settings']['source_environment'] = 'local';`
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-7449_announcement_sync`
* Run `make drush-updb drush-cr`

## How to test
If you tested the `etusivu PR` first, you should already have one or two global announcements created
- Check that you have new external entity type for global announcement
  - `admin/structure/external-entity-types`  
- Check that you have `global announcements` block set up above `announcements block`
  - `/admin/structure/block`  
- Go to any page
  - You should see the global announcements created in etusivu above the site's own announcements.
    - Test with languages and as well, and different announcement types( alert, notice...).
- Go back to `etusivu` and unpublish one or more global announcements
- Go back to original environment and wait for 10 minutes before running `drush cron`
  - unpublished external announcements should disappear 

## Other PRs

[UHF-7449]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-7449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/helfi_node_landing_page/config/install/metatag.metatag_defaults.node__landing_page.yml [hunk 1]
```diff
@@ -5,4 +5,3 @@ id: node__landing_page
 label: 'Content: Landing page'
 tags:
   description: '[node:field_hero:entity:field_hero_desc]'
-
```
Comment (juho-lehmonen) [inline modules/helfi_node_landing_page/config/install/metatag.metatag_defaults.node__landing_page.yml:8]: This config file is probably not related to this PR

File: modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ content:
           show: true
```
Comment (juho-lehmonen) [inline modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml:49]: Are these changes related to this PR?

**Review by juho-lehmonen — CHANGES_REQUESTED** [review]
Works as described, just found a few irrelevant configs.

File: modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml (hunk 2 above)
Comment (rpnykanen) [reply modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml:49]: nope, config export just decided that those strings with empty space need quotes

File: modules/helfi_node_landing_page/config/install/metatag.metatag_defaults.node__landing_page.yml (hunk 1 above)
Comment (rpnykanen) [reply modules/helfi_node_landing_page/config/install/metatag.metatag_defaults.node__landing_page.yml:8]: Not related. drush config export decided that this is important

File: modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml (hunk 2 above)
Comment (rpnykanen) [reply modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml:49]: Non-related. drush export decided to add quoted to the string because of the space between the words.

**Review by juho-lehmonen — APPROVED** [review]
